examples/sqlite.roc

```diff
@@ -0,0 +1,33 @@
+app [main!] { pf: platform "../platform/main.roc" }
+
+import pf.Env
+import pf.Stdout
+import pf.Sqlite
+
+main! = \_args ->
+    db_path = Env.var!? "DB_PATH"
+
+    todo = query_todos_by_status!? db_path "todo"
+
+    Stdout.line!? "Todo Tasks:"
+    List.forEachTry!? todo \{ id, task } ->
+        Stdout.line! "\tid: $(id), task: $(task)"
+
+    completed = query_todos_by_status!? db_path "completed"
+
+    Stdout.line!? "\nCompleted Tasks:"
+    List.forEachTry!? completed \{ id, task } ->
+        Stdout.line! "\tid: $(id), task: $(task)"
+
+    Ok {}
```

Comment:
Richard requested we avoid the `!?` in examples and use `try` exclusively until with have parens and commas.

Reply:
Sad, but ok. I prefer `!?` to `try`, but that is just my person preference.

Reply:
Will update